client/state/selectors/is-account-recovery-reset-options-ready.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Return a boolean value indicating whether the account recovery reset options are fetched and ready to be used.
+ *
+ * @param  {Object} state Global state tree
+ * @return {Boolean}      If the reset options are ready.
+ */
+export default ( state ) => {
+	const resetOptions = state.accountRecovery.reset.options;
+	return null == resetOptions.error && 0 < resetOptions.items.length;
```

Comment:
I cant let go of yoda conditions either :smile: but unless `error` has a chance of being `false` I think `! resetOptions.error` might be easier to read here than the `==` check for null.